account_operating_unit/models/company.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2016 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, fields, models
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    inter_ou_clearing_account_id = fields.Many2one('account.account',
+                                                   'Inter-operating unit\
+                                                   clearing account')
+
+    ou_is_self_balanced = fields.Boolean('Operating Units are self-balanced',
+                                         help="Activate if your company is "
+                                              "required to generate a balanced"
+                                              " balance sheet for each "
+                                              "operating unit.",
+                                         default=True)
```

Comment:
I personally prefer to have the arguments in an indented block instead of aligned at the left to the `(`, since IMO it makes better use of line space. But of course it's just a personal preference.